BDLaetd20 content

thea-m opened the pull request:
Could you please review "only" the description of the content, the physical description is not yet finished :)
I am sure it is still full of typos. I don't know whether anyone wants to check the texts now, if not, we can also write in the table that this still needs to be done (I need a break from this file to see anything in it again).

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    ኀበ፡ ይትነበብ፡ መጽሐፈ፡ ተአምርኪ፡ አክናፈኪ፡ ጸልሊ<hi rend="rubric">፨</hi> ወዘሰማዕነ፡ <del rend="erasure"/>
+                                    በልብነ፡ አብቍሲ<hi rend="rubric">፨</hi> አዕፁቂሁ፡ አልዕሊ<hi rend="rubric">፨</hi> ወሕንባባቲሁ፡ አፅድሊ<hi rend="rubric">፨</hi></incipit>
+                                <explicit>
+                                    <locus target="#1ra"/>
```
f. 1va

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <locus target="#1va"/>
+                                    <hi rend="rubric">ኦእግዝእትነ፡ ቅድስት፡ ድን</hi>ግል፡ 
+                                    በ<hi rend="rubric">፪</hi><hi rend="rubric">ማርያም፡</hi> ማ<sic resp="DR">ረ</sic>ሃም፡ እሙ፡ ለኢየሱስ፡ ክርስቶስ፡ ከሀሊ<hi rend="rubric">፨</hi>
+                                    ኀበ፡ ይትነበብ፡ መጽሐፈ፡ ተአምርኪ፡ አክናፈኪ፡ ጸልሊ<hi rend="rubric">፨</hi> ወዘሰማዕነ፡ <del rend="erasure"/>
```
Sorry for the trivial question, just for the sake of clarity: in one of our previous meeting we discussed about the punctuation signs to be adopted in our catalogue. Is ፨ included too?

thea-m replied:
There was disagreement. @SusanneHummel was in favour of standardising to ።. I maintain that in the case of clearly identifiably ፨ (as I believe is the case here), I would prefer to write this. But it can be either substituted by one search/replace action later or possibly be visualised differently for the printed catalogue.

Gnisci replied:
Does it make a difference when doing a string search?

thea-m replied:
if you search in oxygen, yes, but on the app punctuation either is already disregarded (?) or can be made so by Pietro (as are the homophones)

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                <locus target="#7ra"/>
+                                <hi rend="rubric">ወእነግረክሙ፡ አኃዊነ፡ ጸጋ፡ እግዚአብሔ</hi>ር፡ ዘተውህበ፡ ለቤተ፡ ክርስቲያን፡ ዘመቂዶንያ<hi rend="rubric">፨</hi>
+                                እስመ፡ እምብዙኀ፡ መከራ፡ ሕማሞሙ፡ ፈድፈደ፡ ፍሥሐሆሙ፡ ምስለ፡ ዕመቀ፡ ንዲቶሙ፡ ፈድፈደት፡ ፍሥሐ፡ ብዕሎሙ<hi rend="rubric">፨</hi>
+                                እስመ፡ <gap reason="illegible" unit="chars" extent="1"/>ልየ፡ ሰማዕቶሙ፡ ከመ፡ በአምጣነ፡ ኃይሎሙ<hi rend="rubric">፨</hi>
```
The barely readable letter is ለ, but if we want to indicate that in this condition the letter is mostly illegible, it's ok!

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    አኃዊክሙ፡ አጽንዕዋ፡ ለተፋቅሮ<hi rend="rubric">፤</hi> ወእግዚአብሔር፡ ዘበኵሉ፡ ክብር፡
+                                    ጸውዓክሙ፡ ውስተ፡ ዘለዓለም፡ ስብሐቲሁ፡ በኢየሱስ፡ ክርስቶስ<hi rend="rubric">፨</hi>
+                                    ኅዳጠ፡ ሐሚመ<cb n="b"/>ክሙ፡ ውእቱ፡ ይፈጽም፡ ለክሙ፡ ወያጸንዓክሙ፡ ወያሌብወክሙ፡ ዘሎቱ፡
+                                    ስብሐት፡ ወኃይል፡ ለዓለመ፡ ዓለም፡ አ<hi rend="rubric">፤</hi>
```
አ is an abbreviation of አሜን፡, perhaps this might be encoded with `<supplied/>`. What do you think?

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                <explicit type="supplication" xml:lang="gez">
+                                    <locus target="#13ra"/>
+                                    በረከቱ፡ የሃሉ፡ ምስለ፡ ፍቁሩ፡ <persName ref="PRS12911KidanaK"><hi rend="rubric">ኪዳነ፡ ክርስቶስ፡</hi></persName>
+                                    ለዓለመ፡ ዓለም፡ አሜ፡
```
abbreviation of አሜን፡

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                <explicit type="supplication" xml:lang="gez">
+                                    <locus target="#15rb"/>
+                                    በረከቱ፡ የሃሉ፡ ምስለ፡ ገብሩ፡ <persName ref="PRS12911KidanaK"><hi rend="rubric">ኪዳነ፡ ክርስቶስ፡</hi></persName>
+                                    ለዓለመ፡ ዓለም፡ አ፤
```
abbreviation of አሜን፡

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                <notatedMusic><desc>The hymn included in this work, which is split here in its two parts with
+                                    <ref type="work" corresp="LIT4614Salam#Berhanat"/> on
+                                    <locus from="24ra" to="24rb"/> and <ref type="work" corresp="LIT4614Salam#Negest"/> on
+                                    <locus from="24va" to="24vb"/> contains musical notation throughout.</desc></notatedMusic>
```
comma after `to="24vb"/>`

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <locus from="24va" to="24vb"/> contains musical notation throughout.</desc></notatedMusic>
+                                
+                                <incipit xml:lang="gez">
+                                    <pb n="21r"/><cb n="a"/>
```
Is `<locus from="21ra" to="21rb"/>` omitted here for some reason?

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <hi rend="rubric">ማርያም፡</hi> ወላዲተ፡ አምላክ<hi rend="rubric">፨</hi> ዝንቱ፡ መጽሐፈ፡ ሥርዓት፡ ዘወፅአ፡ እመንበረ፡ ማርቆስ፡ ሐዋርያ፡ እመካነ፡ መዓልቃ፡ ዘምስር<hi rend="rubric">፨</hi>
+ዘአንበሩ፡ መምህራነ፡ ሊቃነ፡ ጳጳሳት፡ ክቡራን፡ መጋብያ<hi rend="rubric">ነ፡ ምሥጢር፡ ርቱዓነ፡ ሃይማኖት። ትትናገሮ</hi>ሙ፡ እግዝእትነ፡ ቅድስት፡ ድንግል፡ በ<hi rend="rubric">፪</hi>ኤ፡ <hi rend="rubric">ማርያም፡</hi> ወላዲተ፡ 
+                                    አ<cb n="b"/><hi rend="rubric">ምላክ፡ ዘልፈ፡ ቦአመ፡ ትትናገሮሙ፡ በሕል</hi>ም፡ ወቦ፡ አመ፡ ትትናገሮሙ፡ በራእይ፡ ወቦ፡ <hi rend="rubric">አመ፡ ትትናገሮሙ፡ በገሃድ፡ እሉ፡ እሙንቱ፡</hi> አባ፡ አብርሃም፡ ወአባ፡ ማቴዎስ፡
+                                    ወአባ፡ ማርዎስ<hi rend="rubric">፨</hi>
```
ማርቆስ፡

thea-m commented:
Thank you, @MassimoVilla ! Could you write a quick comment when you're done with this reviewing session, so that I can add the corrections in a single commit to avoid e-mail overload.

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <locus target="#29rb"/>
+                                    <hi rend="rubric">ወላዲተ፡ አምላክ፡ ማርያም፡ እንበለ፡ ሰብሳ</hi>ብ፡ ወሩካቤ፡ በል<hi rend="rubric">፨</hi>
+                                    ወእምዝ፡ ሰላም፡ ለኪ፡ <hi rend="rubric">ማርያም፡</hi> ኆኅተ፡ ምሥራቅ፡ ዘወልደ፡ ኖሬ፡ ሰላም፡ ለኪ፡ ዓፀድ፡ ዘኤስኬድሬ<hi rend="rubric">፨</hi>
+                                    ሰላም፡ ለኪ፡ ሥርጉት፡ በወራውሬ<hi rend="rubric">፨</hi> ሰ፡ ለ፡ እንተ፡ ወለድኪ፡ ፈጣሬ<hi rend="rubric">፨</hi>
```
ሰ፡ ለ፡ is an abbreviated form of ሰላም፡ ለኪ፡. Do we want to encode this information?

thea-m commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -10,7 +10,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                <title>Taʾammǝra Māryām, Taʾammǝra ʾIyasus, Homily by Jacob of Sarug on the Angel and the Thief</title>
+                <title xml:lang="gez">ተአምረ፡ ማርያም፡, ተአምረ፡ ኢየሱስ፡, ድርሳን፡ ቃሉ፡ ዘቅዱስ፡ ወብጹዕ፡ ያዕቆብ፡ ኤጲስ፡ ቆጶስ፡ ዘሥሩግ፡ በእንተ፡ መልአክ፡ ወፈያታዊ፡</title>
+                <title xml:lang="gez" type="normalized">Taʾammǝra Māryām, Taʾammǝra ʾIyasus, Dǝrsān qālu za-bǝṣuʿ Yāʿqob ʾeṗis qoṗos za-Śǝrug baʾǝnta malʾak wa-fayyātāwi</title>
+                <title xml:lang="en">Hymns, New Testament excerpts, Miracles of Mary, Miracles of Jesus, Homily by the blessed Yāʿqob, bishop of Śǝrug on the Angel and the Thief</title>
```
Clearly, the suggested title format does not work well in cases like this, where works that are not all listed under a known Geez name are contained in the manuscripts. Do you have suggestions?

MassimoVilla replied:
this is a point we should perhaps discuss again in a meeting. Whatever solution we adopt, I fear that a certain degree of incoherence seems unavoidable (like in the solution here proposed). If we choose to omit some works (e.g. here hymns and biblical readings), we should also avoid to do it in some mss but not in others. For the moment we might keep this ms as a good example of this issue and compare it with additional examples in our catalogue.

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    
+                                    <explicit>
+                                        <locus from="52va" to="52vb"/>
+                                        ወደንሀፁ፡ ዓቢየ፡ ድንጋፄ፡ ወለአኩ፡ ኀበ፡ ዓበይተ፡ ሀገር፡ ወመጽኡ፡ ተባዕቶሙ፡ ወአንስቶሙ፡ ወአንከሩ፡ ርእዮሙ፡ ዘንተ፡
```
ወደንገፁ፡

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        ቅድስት፡ ድንግል፡ በ<hi rend="rubric">፪</hi> <hi rend="rubric">ማርያም፡</hi> ወላዲተ፡
+                                        አምላክ። ወሰገዱ<hi rend="rubric">፨</hi> ላቲ፡ <hi rend="rubric">፩፩</hi> አምጣነ፡ ክሂሎቶሙ፡
+                                        ወቀደሰ፡ ውእቱ፡ ቀሲስ፡ ዮሐንስ<hi rend="rubric">፤</hi> በይእቲ፡ ዕለት፡ ወመጠዎሙ፡ እምሥጢሩ፡ ቅዱስ፡ ለመድኃኔ፡
+                                        ዓለም፡ ለመነኮሳት<hi rend="rubric">፨</hi> ወነበረ፡ እምድኅረ፡ዝ፡ <hi rend="rubric">፳</hi> ዓመተ፡
```
I think እምድኅረ፡ዝ፡ should still be typed እምድኅረ፡ ዝ፡ although they are very close (፡ was possibly added later in the middle).

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        ጸሎታ፡ ወበረከታ፡ የሃሉ፡ ወምሕረተ፡ ፍቁር፡ ወልዳ፡ ምስለ፡ 
+                                        ገብራ፡
+                                        <persName ref="PRS12911KidanaK"><hi rend="rubric">ኪዳነ፡ 
+                                            ክርስቶስ፡</hi></persName> ኪዳነ፡  ለዓለመ፡ <pb n="54v"/><cb n="a"/>
```
double space between ኪዳነ፡ and ለዓለመ፡

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        <locus from="54va" to="54vb"/>
+                                        ወሀሎ፡ <hi rend="rubric">፩</hi> ብእሲ፡ ባዕ<cb n="b"/>ል፡ በደሴተ፡ ቌላስይስ፡ ባዕል፡ ጥቀ<hi rend="rubric">፨</hi> 
+                                        ወሖረ፡ ትርኵ፡ በሐመር፡ ከመ፡ ይጽብዖሙ፡ ሰቍዝ። ወተራከብዎ፡ በጽንፈ፡ ባሕር፡ ወነደፍዎ፡
+                                        በ<hi rend="rubric">፲</hi> ወ<hi rend="rubric">፩</hi> አሕፃ፡ 
```
I think there should be no space in በ፲ወ፩አሕፃ፡

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        ወሖረ፡ ትርኵ፡ በሐመር፡ ከመ፡ ይጽብዖሙ፡ ሰቍዝ። ወተራከብዎ፡ በጽንፈ፡ ባሕር፡ ወነደፍዎ፡
+                                        በ<hi rend="rubric">፲</hi> ወ<hi rend="rubric">፩</hi> አሕፃ፡ 
+                                        ወእ<add place="above">ም</add>ሕፃሰ፡ <hi rend="rubric">፩</hi> እምኔሆሙ፡ ነደፎ፡ ዓይኖ፡ ወአልጸቀ፡
+                                        ለመዊት፡ ወኮነ፡ ፅቡረ<hi rend="rubric">፨</hi>  ወእምዝ፡ ይቤሎሙ፡ ለሰብኡ፡ አዕትውኒ፡ ቤትየ፡ ወአተውዎ፡ ቤቶ፡ 
```
double space before ወእምዝ፡

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        <locus from="55va" to="55vb"/>
+                                        ወገብረ፡ ምጽዋተ፡ በዙኃ፡ ለነዳያን፡ ወለምስኪናን፡ ወለቤተ፡ ክርስቲያን፡ ወኀቤሃ<hi rend="rubric">፨</hi> ዘይደሉ<hi rend="rubric">፨</hi> 
+                                        ወሶበ፡ ርእየት፡ እግዝእትነ፡ ቅድስት፡ ድንግል፡ በ<hi rend="rubric">፪</hi> <hi rend="rubric">ማርያም</hi> 
+                                        ወላዲተ፡ አምላክ፡ ቲብዓተ፡ ልቡ፡ ተመልሐ፡ ሐፅ፡ እምዓይኑ፡ ወእምዓይነ፡ ምስል፡ ዘሰምዕ፡ በአሐቲ፡ ሌሊት<hi rend="rubric">፨</hi> 
```
The word ቲብዓተ፡ does not exist. The right expression should be ትብዓተ፡ ልቡ፡ ("with a steady heart" or something similar). It seems to me that a little dash is placed above the letter ቲ

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        ወገብረ፡ ምጽዋተ፡ በዙኃ፡ ለነዳያን፡ ወለምስኪናን፡ ወለቤተ፡ ክርስቲያን፡ ወኀቤሃ<hi rend="rubric">፨</hi> ዘይደሉ<hi rend="rubric">፨</hi> 
+                                        ወሶበ፡ ርእየት፡ እግዝእትነ፡ ቅድስት፡ ድንግል፡ በ<hi rend="rubric">፪</hi> <hi rend="rubric">ማርያም</hi> 
+                                        ወላዲተ፡ አምላክ፡ ቲብዓተ፡ ልቡ፡ ተመልሐ፡ ሐፅ፡ እምዓይኑ፡ ወእምዓይነ፡ ምስል፡ ዘሰምዕ፡ በአሐቲ፡ ሌሊት<hi rend="rubric">፨</hi> 
+                                        ወገብአት፡ ዓይኑ፡ ከመ፡ ካልዕታ<hi rend="rubric">፨</hi>  ወሶበ፡ ርእዩ፡ ሕዝብ፡ ዓይነ፡ ሥዕል፡ አንከሩ፡ ተአምሪሃ፡
```
Double space before ወሶበ፡

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    
+                                    <incipit type="inscriptio" xml:lang="gez">
+                                        <locus target="#56ra"/>
+                                        <hi rend="rubric">ተአምሪሃ፡ ለእግዝእ</hi>ትነ፡ ቅድስት፡ ድን <hi rend="rubric">ግል፡ በ፪ ማርያም፡
```
no space in `ድን<hi rend="rubric">ግል፡`. I also think there should be no space in በ፪ማርያም፡

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    
+                                    <explicit type="supplication" xml:lang="gez">
+                                        <locus from="60rb" to="60va"/>
+                                        ጸሎታ፡ ወበረከታ፡  ወምሕረተ፡ ፍቁር፡ ወ<pb n="60v"/><cb n="a"/>ልዳ፡ የሃሉ፡
```
Double space between ወበረከታ፡ and ወምሕረተ፡

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    </incipit>
+                                    
+                                        <incipit xml:lang="gez">
+                                            <locus target="#60vb"/> ወሀሎ፡ <hi rend="rubric">፩</hi>
```
no space before ወሀሎ፡

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                            <locus target="#60vb"/> ወሀሎ፡ <hi rend="rubric">፩</hi>
+                                            ብእሲ፡ መስተጋድል፡ ፈድፋደ፡ በከዊነ፡ ስምዕ፡ በእንተ፡ ስመ፡ ክርስቶስ፡ ዘስሙ፡
+                                            ጊዮርጊስ፡ ሐዲስ<hi rend="rubric">፨</hi> ወአሐተ፡ ዕለተ፡ እንዘ፡ ሀሎ፡
+                                            ተጽዒሮ፡ እምብዝኃ፡ ኵነኔ፡ ኖመ፡ ወበ<hi rend="rubric">፲</hi> ወ<hi rend="rubric">፩</hi>
```
I think we might discuss in a future meeting how to consistently encode the sequence numeral+word (or numeral+ወ+numeral+word) in those cases where no colon ፡ is written between them.

thea-m replied:
Yes!

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    
+                                    <explicit type="supplication" xml:lang="gez">
+                                        <locus target="#61rb"/>
+                                        ጸሎታ፡ ወበረከታ፡  ወምሕረተ፡ ፍቁር፡ ወልዳ፡ የሃሉ፡ ምስለ፡ ገብራ፡
```
double space between ወበረከታ፡ and ወምሕረተ፡

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        አስተዓፀበ፡ ወሖረ፡
+                                        <pb n="66r"/><cb n="a"/>ምስለ፡ ኵሉ፡ ሕዝብ፡ ወሰገደ፡ ቅድመ፡ ሥዕላ፡ ለእግዝእትነ፡ ቅድስት፡ ድንግል፡ 
+                                        በ<hi rend="rubric">፪</hi> <hi rend="rubric">ማርያም፡</hi>
+                                        ወላዲተ፡ አምላክ፡ ወሰብሖ፡ ለእግዚአንሔር፡ ወአእኰታ፡ ለእግዝእትነ፡ ቅድስት፡ ድንግል፡ 
```
ለእግዚአብሔር፡

Gnisci commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                            <locus target="#117rb"/>
+                            <colophon xml:id="coloph1" xml:lang="gez">
+                                <note xml:lang="en">The colophon states that this manuscript was commissioned by 
+                                    <persName ref="PRS12911KidanaK" role="sponsor"/> who donated it to a church of 
```
"to a church identified as" ?

Gnisci commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -51,11 +3473,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             <supportDesc>
                                 <support>
                                     <material key="parchment"/>
+                                    <p>Some folia have small holes, presumably from parchmenting: <locus
```
"parchmenting"is this the right term?

SusanneHummel replied:
I know that Sean Winslow is using this term. Therefore I also use it :)

Gnisci replied:
I am not convinced, a search on scholar provides less than a dozen attestations: https://scholar.google.it/scholar?hl=en&as_sdt=0%2C5&q=%22parchmenting%22+manuscript&btnG=

SusanneHummel replied:
We can use "parchment making" as in ligatus (https://www.ligatus.org.uk/lob/concept/4962)

Gnisci commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -51,11 +3473,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <p>Some folia have small holes, presumably from parchmenting: <locus
+                                        target="#15 #50 #85"/>
+                                    </p>
+                                    <p>Small tears amended on<locus
```
"repaired" rather than "amended"

Gnisci commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -159,29 +3591,82 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                           
+                           <seg type="ink">Black, red.</seg>
+                                <seg type="rubrication">
+                                    Incipits (one word to one or several lines, consecutive or alternating with black lines in different groupings), 
```
Could/should this be more discursive? Question to all of you

Gnisci commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -159,29 +3591,82 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                
+                                <item xml:id="a2">
+                                    <locus target="#5r"/>
+                                    <desc type="Inventory">Further list of objects, separated from
```
"An additional list of objects,"

Gnisci commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -159,29 +3591,82 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             <binding xml:id="binding">
-                                <decoNote xml:id="b1"></decoNote>
+                                <decoNote xml:id="b1">Two<material key="wood">wooden</material>
+                                    boards previously fully covered with reddish-brown blind-tooled <material
```
"that were previously fully covered"

Gnisci commented:
This was a real tour-de-force Thea, OMG, thank you!

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -159,29 +3591,82 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         <dim type="right">35</dim>
                                         <dim type="intercolumn">14</dim>
                                     </dimensions>
+                                    <ab type="ruling" subtype="pattern">Ruling pattern:  1A-1A-1A1A/0-0/0-0/C.</ab>
```
double space before the pattern

MassimoVilla commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -159,29 +3591,82 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                
+                                <item xml:id="a3">
+                                    <locus target="#7r"/>
+                                    <desc type="OwnershipNote">Note stating that this manuscript belonged to 
```
I would add that the note is written in pale black ink in the upper corner of the folio.

thea-m commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <incipit>
+                                        <locus from="37ra" to="37rb"/>
+                                        <hi rend="rubric">፫ተአምሪሃ፡ ለእግዝእትነ፡ 
+                                            ቅድስት።</hi> ድንግል፡ በክልኤ፡  <hi rend="rubric">ማርያም፡</hi> ወላዲተ፡ አምላክ<hi rend="rubric">፨</hi>
```
I corrected it for our aesthetics (it also annoys me to see this elsewhere), but actually from the XML point of view, one space or more are completely irrelevant.

SusanneHummel replied:
Is the space also irrelevant for the later transmission to the pdf?

thea-m replied:
Yes

thea-m commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    
+                                    <explicit xml:lang="gez">
+                                        <locus from="74vb" to="75ra"/>
+                                        ወንድቅ፡ ዘተሠጥቀ፡ ተላጸቀ፡ ወኮነ፡ ከመ፡ ቀዳሚ፡ ወባሕቱ፡ ንስቲት፡ መጠነ፡ ሙባአ፡ እድተረፈ፡ ለ<!--mut-->ተዛካር፡ ወዘንተ፡
```
I think he wrote ሊ before correcting to ለ, but didn't yet open an issue on how to mark this up

thea-m commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <locus target="#84va"/>
+                                    ጸሎታ፡ ወበረከታ፡ የሃሉ፡ ምስለ፡ ገብራ፡
+                                    <persName ref="PRS12964Gorgoryos"><hi rend="rubric">ጎርጎርዮስ፡</hi></persName> ለዓለመ፡
+                                    ዓለም፡ አሜን<hi rend="rubric">፨</hi> <hi rend="rubric">፨</hi> <hi rend="rubric">፨</hi>
```
It's broadly part of my dissatisfaction with encoding partial and complete rubrication the same way and the ensuing generalisation in encoding rubrication. But no, both would be fine here!

thea-m commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    ወአሐተ፡ ዕለተ፡ እንዘ፡ ይነውም፡ ምስለ፡ ረድኡ፡ ውስተ፡ በአቱ፡ ወካልእኒ፡ መነኮስ<hi rend="rubric">፨</hi>
+                                </incipit>
+                                
+                                <!--this is the text of the entire miracle-->
```
Yes, the old question...did you find the q regulation anywhere in the guidelines? I think it's only documented for additions. I still wanted to open an issue.

thea-m commented on OxfordBodleian/Juel-Jensen/BDLaethd20.xml:
```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                            <locus target="#117rb"/>
+                            <colophon xml:id="coloph1" xml:lang="gez">
+                                <note xml:lang="en">The colophon states that this manuscript was commissioned by 
+                                    <persName ref="PRS12911KidanaK" role="sponsor"/> who donated it to a church of 
```
the schema has:
patron: the commissioner of a ms or an item
sponsor: the person paying for a ms (possibly not the same as
                                        donor/commissioner)
Commissioner as a value does not exist anymore.
I've changed here to "patron" and will list the other values (sponsor and donor) in the listPers

thea-m commented:
thanks everyone! Everything I did not comment indivudally I completely agree with and corrected now. 
For me personally, it would also be fine if corrections such as @MassimoVilla 's very well spotted typo corrections would be corrected directly in the file (I don't think it takes longer for the reviewer than to write a comment?), but the current way is also completely ok.

Gnisci commented:
> 
> 
> thanks everyone! Everything I did not comment indivudally I completely agree with and corrected now.
> For me personally, it would also be fine if corrections such as @MassimoVilla 's very well spotted typo corrections would be corrected directly in the file (I don't think it takes longer for the reviewer than to write a comment?), but the current way is also completely ok.

Presumably we can also remove double spacing through a search and replace?

thea-m commented:
> Presumably we can also remove double spacing through a search and replace?

Hmm, I cannot, but there must be way...
But apart from the aesthetics, it's really not relevant.

thea-m commented:
In my last commit, I moved the text of the two miracles I copied in their entirety to `<div>`. @PietroLiuzzo now we have the old "problem" (maybe not?) of a pb at the beginning of the text, when the copied text does not begin at the top of the page. I added `<gap>`, which is also misleading, because I have a `@corresp` in the div to the respective msItem, of whose text there is no gap...

PietroLiuzzo commented:
can you add an `lb` ? e.g. if your copied text start at line 5:
` <pb n="90r"/><cb n="a"/><lb n="5"/>`
this way you avoid the false gap, the note and all the rest.

PietroLiuzzo commented:
you will also be able to refer to this in locus with e.g. from="90ra5"

thea-m commented:
Thank you, that's good! Now I have to ask - there is no need to refer to it with the line number in locus, right, just a possibility?

PietroLiuzzo commented:
just a possibility. you can, if you so wish, you do not have to.

thea-m commented:
> > Presumably we can also remove double spacing through a search and replace?
> 
> Hmm, I cannot, but there must be way...
> But apart from the aesthetics, it's really not relevant.

Obviously, there is one...
![image](https://user-images.githubusercontent.com/19650382/85222633-132b0a00-b3bd-11ea-920e-6e230ad27bc3.png)
Simple the "Format and Indent" button 😆

PietroLiuzzo commented:
let me  share a word of caution about format and indent. this is 98% of the times perfectly fine, however, you may incur into spaces which you did not intend to have. the "double spaces", depending which kind of space, should definitely not be that relevant, because one space or more spaces are the same in an XML file. only space or no space is a relevant distinction. If the visualisation has extra/ double spaces I can have a look at if they can be normalised, in which case please open an issue with proper examples. thanks

thea-m commented:
Could you give us an example of some problematic results?

thea-m commented:
I think that for the purpose of a first PR this can be merged?
